cmd/convert.go

```diff
@@ -46,6 +46,7 @@ var convertCmd = &cobra.Command{
 			ToStdout:               ConvertStdout,
 			CreateChart:            ConvertChart,
 			GenerateYaml:           ConvertYaml,
+			GenerateJson:           ConvertJson,
```

Comment:
After doing some iterations on trying to get rid of ConvertYaml it feels like it's only making code more convoluted in terms of making checks and validation, I think we can get rid of ConvertYaml once we decide we wanna get rid of the flag.